### ISSUE TITLE
Add in a `GENERAL_INFO.md` document?

### ISSUE THREAD

**Issue by jGaboardi** [pysal/submodule_template#37]
We should maybe add some sort of file in the top level for general information, etc. like:

* `versioneer` usage and set up
* `pre-commit` set up
* GHA and secrets information
* links to Wikis and other repos for demos
* other stuff

Maybe a file like `GENERAL_INFO.md`, where we make it clear that the file should be deleted when setting up an actual repo.

**Comment by knaaptime** [comment]
yeah, and it should probably replace https://pysal.org/docs/devs/ with the latest conventions (and maybe we leave this kinda developer documenting to here instead of the website so we dont get competing resources?)

**Comment by jGaboardi** [comment]
> yeah, and it should probably replace https://pysal.org/docs/devs/ with the latest conventions (and maybe we leave this kinda developer documenting to here instead of the website so we dont get competing resources?)

Seems like a reasonable idea.

**Comment by martinfleis** [comment]
I would personally update https://pysal.org/docs/devs/ with such an info rather than creating another place for it.

**Comment by knaaptime** [comment]
Agree we should have a single source, but i'd rather re-canonize it here. To me, the website feels like a place for users more than developers, whereas the submodule contract is where it makes more sense to house the 'dev set up' instructions. 

Right now both the submodule template and the website contain competing instructions, so i vote here rather than there. Tbh, i forget the website instructions even exist

**Comment by martinfleis** [comment]
I don't mind where is it, but agree that there should be only one source of truth. Could be here.
